doc/hero_images.md

```diff
@@ -0,0 +1,81 @@
+#### circuitpython.org Documentation Pages
+
+## Documentation for Hero Images on the circuitpython.org homepage
+
+The hero, or main, image is the picture seen when first arriving on the circuitpython.org website. As this is 
+a major part of the look and feel of the site, care should be taken to ensure the image used fits established 
+specifications and aesthetics. 
+
+### Hero Image Specifications
+
+- Hero images are stored in the /assets/images/heros directory in the repo.
+- The images can be .png or .jpg files.
+- The image size must be 1440 pixels by 800 pixels.
+- The image ideally will be composited such that overlay text for the page blends into the style of the image.
+
+### Where the Hero Image is specified in the Site Layout
+
+The hero image on circuitpython.org may be changed by editing a value in the _config.yml file.
+
+The _config.yml file is located in the root directory of the circuitpython.org repo on GitHub.
+
+The typical contents (truncated after a number of lines) is similar to:
+
+  title: CircuitPython
```

Comment:
Looks like the formatting here is not proper when it renders - there are no newlines, it renders as a paragraph.

Reply:
Thanks, yes I thought I had tabs in, changed to 4 spaces and markdown formats right. I also bolded the filenames. Ready again I hope